core-web/libs/block-editor/src/lib/extensions/ai-content-prompt/ai-content-prompt.component.html

```diff
@@ -12,13 +12,25 @@
             class="ai-content__form"
             [formGroup]="form"
             (ngSubmit)="store.generateContent(form.value.textPrompt)">
-            <textarea
-                class="ai-content__generated-text"
-                *ngIf="vm.status !== ComponentStatus.LOADING; else skeleton"
-                [placeholder]="'block-editor.extension.ai-content.placeholder' | dm"
-                [readOnly]="true"
-                formControlName="generatedText"
-                pInputTextarea></textarea>
+            <ng-container
+                [ngTemplateOutlet]="
+                    vm.status === ComponentStatus.LOADING
```

Comment:
I think a templateOutlet is too hard for this case
And the comparation on this way is hard to read
Maybe can replace with

```
@if(vm.status === ComponentStatus.LOADING){
} @else if (vm.generatedContent[vm.activeIndex]?.error) {

} @else {

}
```

Reply:
done